julia/Related/src/Related.jl

```diff
@@ -41,9 +41,11 @@ function fastmaxindex!(xs::Vector{T}, topn, maxn, maxv) where {T}
 end
 
 function related(posts)
-    Ts = (UInt8, UInt16, UInt32, UInt64)
-    i = findfirst(T -> length(posts) < typemax(T), Ts)
-    return related(Ts[i], posts)
+    for T in (UInt8, UInt16, UInt32, UInt64)
+        if length(posts) < typemax(T)
```

Comment:
This will no longer be allowed. Use a static size that abides by the rules. ie: uint32.

Reply:
we need to also make a change to the non-concurrent version I guess.

Reply:
did it for the concurrent version